packages/gsm8k/src/aviary/envs/gsm8k/env.py

```diff
@@ -111,10 +111,15 @@ async def step(
                 total_reward += reward
                 any_done |= done
 
-            return response_msgs + invalid_response_msgs, total_reward, any_done, False
+            return (
+                cast(Messages, response_msgs + invalid_response_msgs),
```

Comment:
I want to fix these type errors in another PR, can we use `type: ignore` for now over `cast`